Modules/Filtering/DisplacementField/include/itkComposeDisplacementFieldsImageFilter.hxx

```diff
@@ -60,7 +60,7 @@ template <typename InputImage, typename TOutputImage>
 void
 ComposeDisplacementFieldsImageFilter<InputImage, TOutputImage>::BeforeThreadedGenerateData()
 {
-  VectorType zeroVector(0.0);
+  VectorType zeroVector{};
```

Comment:
I bet many of these `zeroVector` variables could be `const`.  (Perhaps for another day.)

Reply:
I bet many of these `zeroVector` variables could even be `constexpr`  😸 I still hesitate about including such changes with this commit. Maybe we could run a tool to constify (or "constexprify") local variables automatically, independent of this PR....?

Reply:
@N-Dekker  Let me know if you find such a "constify" tool.  I have a few hundred projects I'd love to run it on :).